src/Plotting/paraview.jl

```diff
@@ -0,0 +1,49 @@
+function Base.convert(::Type{<:VTKPolyhedron}, X::LazySet; shift=0)
```

Comment:
I suggest to use a different function. `convert` should have only one result.

Reply:
ok. i restructured a bit the PR since the triangulation part is independent of WriteVTK.jl